src/components/Foods.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import foodSeed from '../foods.json'
+import FoodBox from './FoodBox'
+import AddFood from './AddFood'
+import TodayFood from './TodayFood'
+
+
+class Foods extends React.Component {
+    constructor(props){
+        super(props)
+        this.state ={
+            foodList: foodSeed,
+            filteredFoods: foodSeed,
+            formShow: false,
+            TodayFoods: []
+        }
+    }
+
+    formAppear = () => {
+        this.setState({
+            formShow: true
+        })
+    }
+
+    handleAdd = (e) => {
+        e.preventDefault()
+        let foodName        = e.currentTarget.foodName.value
+        let foodCalories    = e.currentTarget.foodCalories.value
+        let foodImage       = e.currentTarget.foodImage.value
+        let foodQuantity    = e.currentTarget.foodQuantity.value
+        let newFood = [
+            {name:      foodName, 
+            calories:   foodCalories,
+            image:      foodImage,
+            quantity:   foodQuantity   
+            }, ...this.state.foodList]
+
```

Comment:
just for readability, this is the common way to write objects nested inside arrays.

```
let newFood = [
      {
        name: foodName,
        calories: foodCalories,
        image: foodImage,
        quantity: foodQuantity,
      },
      ...this.state.foodList,
    ];
```